frontend/tests/unit/services/transactions/request-manager.spec.ts

```diff
@@ -331,8 +330,7 @@ describe('request-manager', () => {
     it('throws an error if the transaction receipt is undefined', async () => {
       const transactionHash = getRandomString();
 
-      const contract = mockGetContract();
-      contract.interface.parseLog = vi.fn().mockReturnValue({ args: { requestId: '1' } });
```

Comment:
There was no need of mocking the `parseLog` here as this test case was not reaching that scope due to the condition checks.